src/MatchInterface.php

```diff
@@ -15,13 +15,13 @@ interface MatchInterface
      *
      * @return string
      */
-    public function getName();
+    public function getName(): string;
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!